src/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-alert */
+import './styles.css';
+import { submitScore, fetchLeaderboard } from './functionality.js';
+
+// Event listener for the "Submit" button
+document.getElementById('submit-button').addEventListener('click', () => {
+  const nameInput = document.getElementById('name-input');
+  const scoreInput = document.getElementById('score-input');
+  const name = nameInput.value.trim();
+  const score = parseInt(scoreInput.value, 10);
+
+  if (name === '') {
+    alert('Please enter your name.');
+    return;
+  }
+
+  if (Number.isNaN(score)) {
+    alert('Please enter a valid score.');
+    return;
+  }
+
+  submitScore(name, score);
+});
+
+// Event listener for the "Refresh" button
+document.getElementById('refresh-button').addEventListener('click', () => {
+  fetchLeaderboard();
```

Comment:
Good job working on the project requirements. Its against microverse policy to disable eslint or any other linters because linters help us write clean code. it would be nice if you enable eslint.